src/pineko/theory.py

```diff
@@ -22,6 +22,21 @@
 logger = logging.getLogger(__name__)
 
 
+def _load_grids_from_nnpdf(dataset_name):
+    """Load the list of fktables/grids necessary from a NNPDF dataset."""
+    # Hide the validphys import to avoid unnecessary failures
+    from validphys.commondataparser import EXT
+    from validphys.datafiles import legacy_to_new_map
+    from validphys.loader import Loader
```

Comment:
will they eventually move to `nnpdf_data`? (since they did not yet in https://github.com/NNPDF/nnpdf/pull/2009 ? )

Reply:
It's already merged to master.
However, at the moment this will rely on vp since I'd rather use the `Loader` that does various checks.